packages/opentelemetry-resources/test/Resource.test.ts

```diff
@@ -287,6 +287,150 @@ describe('Resource', () => {
     });
   });
 
+  describe('schema URL support', () => {
+    it('should create resource with schema URL', () => {
+      const schemaUrl = 'https://example.test/schemas/1.2.3';
+      const resource = resourceFromAttributes({ attr: 'value' }, { schemaUrl });
+
+      assert.strictEqual(resource.getSchemaUrl?.(), schemaUrl);
+    });
+
+    it('should create resource without schema URL', () => {
+      const resource = resourceFromAttributes({ attr: 'value' });
+
+      assert.strictEqual(resource.getSchemaUrl?.(), undefined);
+    });
+
+    it('should retain schema URL from base resource when other has no schema URL', () => {
+      const schemaUrl = 'https://opentelemetry.test/schemas/1.2.3';
+      const resource1 = resourceFromAttributes(
+        { attr1: 'value1' },
+        { schemaUrl }
+      );
+      const resource2 = resourceFromAttributes({ attr2: 'value2' });
+
+      const mergedResource = resource1.merge(resource2);
+
+      assert.strictEqual(mergedResource.getSchemaUrl?.(), schemaUrl);
+    });
+
+    it('should retain schema URL from the resource that has it when merging', () => {
```

Comment:
Looks like you might have a duplicated test case between this and the next case.